tests/workflows/protocols/pw/test_relax.py

```diff
@@ -63,3 +63,12 @@ def test_spin_type(fixture_code, generate_structure):
         parameters = namespace['pw']['parameters'].get_dict()
         assert parameters['SYSTEM']['nspin'] == 2
         assert parameters['SYSTEM']['starting_magnetization'] == {'Si': 0.1}
+
+
+@pytest.mark.parametrize('relax_type', RelaxType)
```

Comment:
Just so I understand: since `RelaxType` is an `Enum`, and hence an iterable, `pytest.mark.parametrize` will test each of the `RelaxType`'s defined in the class?

Reply:
[Uhuh](https://media.giphy.com/media/l41m35Z4QtUJgVEdy/giphy.gif)